java-checks/src/main/java/org/sonar/java/checks/CheckList.java

```diff
@@ -20,7 +20,7 @@
 package org.sonar.java.checks;
 
 import com.google.common.collect.ImmutableList;
-
+import java.util.List;
```

Comment:
I set up my Eclipse using the doc [https://github.com/SonarSource/sonar-developer-toolset], file sonar.importorder. Maybe I didn't import it correctly.